interface/portrait_properties/00_portrait_properties.txt

```diff
@@ -4770,4 +4770,48 @@
 			trait = kiljaeden_trait
 		}
 	}
+}
+# Warcraft
+# p28 vrykul
+28 = {
+	0 = {
+		factor = 1
+		trigger = {
+			NOT = {
+				has_portrait_property = { layer = 28 index = 1 }
+				has_portrait_property = { layer = 28 index = 2 }
+				has_portrait_property = { layer = 28 index = 3 }
+			}
+		}
```

Comment:
`has_portrait_property` is bad. Don't use it.
Like if I remove `ymiron_trait` trait, I'll keep looking like Ymiron. I have the 1 index and can't switch to 0 because of these conditions.
```suggestion
```